engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/marshall/ProcessMarshallerSpec.scala

```diff
@@ -59,50 +89,41 @@ class ProcessMarshallerSpec extends FlatSpec with Matchers with OptionValues wit
     result should equal(Some(process))
   }
 
-  it should "omit additional fields" in {
-    val withAdditionalFields =
-      """
-        |{
-        |    "metaData" : { "id": "custom", "typeSpecificData": { "type" : "StreamMetaData", "parallelism" : 2 }, "additionalFields": { "description": "process description"} },
-        |    "exceptionHandlerRef" : { "parameters" : [ { "name": "errorsTopic", "expression": { "language": "spel", "expression": "error.topic" }}]},
-        |    "nodes" : [
-        |        {
-        |            "type" : "Source",
-        |            "id" : "start",
-        |            "ref" : { "typ": "kafka-transaction", "parameters": [ { "name": "topic", "expression": { "language": "spel", "expression": "in.topic" }}]},
-        |            "additionalFields": { "description": "single node description"}
-        |        }
-        |    ]
-        |}
-      """.stripMargin
-
-    val withoutAdditionalFields =
-      """
-        |{
-        |    "metaData" : { "id": "custom", "typeSpecificData": { "type" : "StreamMetaData", "parallelism" : 2 }},
-        |    "exceptionHandlerRef" : { "parameters" : [ { "name": "errorsTopic", "expression": { "language": "spel", "expression": "error.topic" }}]},
-        |    "nodes" : [
-        |        {
-        |            "type" : "Source",
-        |            "id" : "start",
-        |            "ref" : { "typ": "kafka-transaction", "parameters": [ { "name": "topic", "expression": { "language": "spel", "expression": "error.topic" }}]}
-        |        }
-        |    ]
-        |}
-      """.stripMargin
+  it should "not marshall process additional fields" in {
+    val process = EspProcessBuilder
+      .id("process1")
+      .additionalFields(TestAdditionalFields(field1 = "additional field 1", field2 = 2))
+      .exceptionHandler()
+      .source("a", "")
+      .processorEnd("d", "dService", "p1" -> "expr3")
+
+    val result = marshallAndUnmarshall(process)
 
+    result.value.metaData.additionalFields shouldBe 'empty
```

Comment:
It doesn't seem right at first glance. Add a comment if this is intended behaviour.

Reply:
We decided to rework the solution.